deepinv/models/unet.py

```diff
@@ -104,12 +108,16 @@ def conv_block(ch_in, ch_out):
                         bias=bias,
                         padding_mode="circular" if circular_padding else "zeros",
                     ),
-                    BFBatchNorm2d(ch_out, use_bias=bias),
+                    BFBatchNorm2d(ch_out, use_bias=bias)
+                    if biasfree
+                    else nn.BatchNorm2d(ch_out),
                     nn.ReLU(inplace=True),
                     nn.Conv2d(
                         ch_out, ch_out, kernel_size=3, stride=1, padding=1, bias=bias
                     ),
-                    BFBatchNorm2d(ch_out, use_bias=bias),
+                    BFBatchNorm2d(ch_out, use_bias=bias)
```

Comment:
I had removed the BF batch norm from Mohan et al since it often produces NaNs during training. Unless you found a solution to this, I wouldn't add this modle

Reply:
@tachella This PR means UNet defaults to nn.BatchNorm2d but if the user really wants, they can pass `batch_norm=biasfree` to use BF batch norm to maintain compatibility.

The BF batch norm did give me problems with NaNs during training, yes, btw.

Reply:
ok, I agree. 

Could you please add a link to the arxiv paper when mentioning the biasfree batchnorm option in the params (and maybe a warning about the nans using [sphinx's warning](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html))?

Reply:
Done, thanks @tachella

Reply:
thanks!